openapi.yaml

```diff
@@ -385,6 +405,9 @@ components:
         title:
           type: string
           maxLength: 128
+        bio:
```

Comment:
bio & partners should be available in WorkspaceCreateRequest & WorkspaceUpdateRequest

Reply:
> @angeljgomezc almost there -- just need to add two test cases:
> 
> 1. workspace create with bio & partners
> 2. workspace update with bio & partners

can you point me where? or is it done already?